src/App.vue

```diff
@@ -1,4 +1,10 @@
 <script setup lang="ts">
+import {onMounted} from "vue"
+
+onMounted( async() => {
+  (await import("./lib/database/official/index")).default()
```

Comment:
This should get a comment stating what this does, as its not really apparent from the import (this starts the trick / combo migration process)